affordance_util/include/affordance_util/affordance_util.hpp

```diff
@@ -254,6 +291,24 @@ Eigen::MatrixXd compose_cc_model_slist(const RobotDescription &robot_description
  space-frame name, joint_names, and tool name
  */
 RobotConfig robot_builder(const std::string &config_file_path);
+/**
+ * @brief Given a file path to a URDF file containing robot information, a ref frame, a base joint, and an ee_frame,
```

Comment:
also optionally, tool location. Clarify that the default behavior is to build the robot with the EE link as the tool, but it is encouraged to provide a tool location that represents the center of the robot palm.